03_reverseString/reverseString.js

```diff
@@ -0,0 +1,5 @@
+const reverseString = function(str) {
```

Comment:
No, this is wrong. I think something got messed up, including some of my work from doing this. Investigating further, may delete this all and start over.

Reply:
Okay, I now understand this better. The original renaming took place over 3 commits, not 2:
* https://github.com/TheOdinProject/javascript-exercises/commit/55cfb173d8da273146a5c598cb8ea3e9760831fa
* https://github.com/TheOdinProject/javascript-exercises/commit/61f38bf60cedaf210ddd067c624a086e0f2c7a71
* https://github.com/TheOdinProject/javascript-exercises/commit/3236ae156ccc536c34435b134e6232d1f930bc3c

And the 2nd of those commits did what I thought I had done, which was it inadvertently checked in a solution to one of the exercises. The third commit attempted to undo that.

But applying all of that now on the solutions branch, the end result is that the actual solution disappeared. I will fix that in another commit.